src/components/RegistrationSuccessMessage.js/index.js

```diff
@@ -0,0 +1,48 @@
+import React, { memo } from 'react'
```

Comment:
.js in folder name is a little bit confusing

Reply:
This slipped in accidentally, removed it